connect/amazon/ses/client/build.gradle.kts

```diff
@@ -10,6 +10,7 @@ plugins {
 
 dependencies {
     api(project(":http4k-connect-amazon-core"))
+    implementation("com.sun.mail:javax.mail:_")
```

Comment:
Optional.  Includes helper to generate a `RawMessagebase64` from a `javax.mail.Message`.

Reply:
Wasn't javax namespace deprecated? I think com.sun.mail:jakarta.mail is now preferred.

Reply:
@aharin Good point.  I've also added optional support for jakarta.  Jakarta mail doesn't like when javax mail is on the same classpath, so javax mail is now added as `compileOnly`, and can't have test coverage.

Reply:
Q - why would we use javax mail at all if it's deprecated now? Shouldn't we just use jakarta as that's the future? :)

Reply:
For the same reason we still support javax servlets? :shrug: I'll concede it's much easier to upgrade to jakarta mail than to jakarta servlets though.  It's just an optional dependency for a small convenience function, but I don't mind removing it, if you want.

Reply:
Nah - the only reason we've still got javax (which arguably should have been removed as a part of v6) was for backwards compatibility.,This is new, so I say we just ditch it.

Reply:
Ok, done

Reply:
👍